resources/views/dashboard/regulated-organization.blade.php

```diff
@@ -22,6 +22,11 @@
                 @endif
             @endcan
         @endcan
+        @if (!$user->oriented_at)
```

Comment:
Need to check the Regulated Organization. Try:
```suggestion
        @if (!$user->regulatedOrganization->oriented_at)
```